test/integration/runonce_duration_test.go

```diff
@@ -0,0 +1,101 @@
+// +build integration
+
+package integration
```

Comment:
Yup I'm assuming test-integration.sh will run a separate process for each test method. (which afaik it does)